test/amperize.test.js

```diff
@@ -561,7 +561,7 @@ describe('Amperize', function () {
         it('can handle <img> tag without src and does not transform it', function (done) {
             amperize.parse('<img><//img><p>some text here</p>', function (error, result) {
                 expect(result).to.exist;
-                expect(result).to.be.equal('<img><p>some text here</p>');
+                expect(result).to.be.equal('<img><!--/img--><p>some text here</p>');
```

Comment:
Not sure why the result of this test changed?

Reply:
@nkeynes That came in with the bump from htmlparser2 from version 4 to version 8, but the change exists at version 5. The release notes for 5 are here: https://github.com/fb55/htmlparser2/releases/tag/v5.0.0

I'm unsure which commit or PR caused the change in that release, but I appears to be doing the right thing anyway where it was probably a bug before? It's changing `<//img>` to `<!--/img-->. A double slash at the start of a HTML tag is non-compliant HTML anyway from what I have dug into and the new version of the parser must be guessing it's someone trying to add a HTML comment, hence the new conversion.

I think the right solution here is to remove the `<//img>` tag from line 562 entirely as it has nothing to do with the test it is in anyway. I'll do that now and keep the version of htmlparser2 at 8.